pkg/virt-api/webhooks/validating-webhook/admitters/vmrestore-admitter.go

```diff
@@ -182,13 +182,8 @@ func (admitter *VMRestoreAdmitter) Admit(ar *admissionv1.AdmissionReview) *admis
 func (admitter *VMRestoreAdmitter) validateCreateVM(field *k8sfield.Path, namespace, name string) ([]metav1.StatusCause, *types.UID, error) {
```

Comment:
Re: this comment, it looks like we have to do more work validation here:  https://github.com/kubevirt/kubevirt/pull/7318#issuecomment-1077825120

If the target exists, have to validate that the snapshot is a snapshot of the target and not a different VM

Reply:
I've had a follow-up comment [here](https://github.com/kubevirt/kubevirt/pull/7318#issuecomment-1086785444).

As explained there, I still think we should not disallow it.

Reply:
Thanks, this is now validated by validating webhooks.